control/init.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
+if ! hash virtualenvwrapper.sh 2>/dev/null; then
+    echo "Please install virtualenvwrapper and make sure it is in your PATH"
+    echo ""
+    echo "  sudo apt-get install git python-dev python-pip"
+    echo "  sudo pip install virtualenv virtualenvwrapper"
+    return 1
+fi
+
+source virtualenvwrapper.sh
+
+if [ ! -d ~/.virtualenvs/ansible ]; then
+    echo "Creating ansible virtualenv..."
+    mkvirtualenv ansible
+    pip install -r ~/commcarehq-ansible/ansible/requirements.txt
+else
+    workon ansible
+fi
+
+if ! [ -L ~/commcarehq-ansible/config -o -d ~/commcarehq-ansible/config ]; then
+    # TODO setup permissions (add user to group?) to allow write to config repo
+    #su ansible -c "usermod -a -G ansible $USER"
+    ln -s /home/ansible/commcarehq-ansible/config ~/commcarehq-ansible/config
```

Comment:
Poll: should we...
- Fiddle with permissions and try to make it so everyone can modify and commit to a single central git repo at `/home/ansible/commcarehq-ansible/config`. This is very similar to what we are doing now. Keeping the permissions correct so everyone can write could be tricky. Vote with 🎉 
- Put a central bare repo at `/home/ansible/commcarehq-ansible-config.git` or some similar path and make a separate clone of that repo in each dev account. Typical git collaboration model (pull, edit, commit, push), and does not require special permissions. Vote with ❤️ 

@dimagi/team-commcare-hq please vote by adding a reaction to this message. Add another comment if you have a better idea.